.travis.yml

```diff
@@ -65,3 +70,12 @@ jobs:
       install:
         - .travis/install_conda.sh "Miniconda3-latest-MacOSX-x86_64.sh"
       script: .travis/conda_build_and_deploy.sh
+    - name: "conda linux python3.6"
+      if: type = cron AND branch = master
+      git:
+        depth: false
+      env:
+        - PATH="$HOME/miniconda/bin:$PATH"
+      install: .travis/install_conda.sh "Miniconda3-latest-Linux-x86_64.sh"
+      script:
+        - .travis/conda_build_and_deploy.sh --python 3.6
```

Comment:
We should probably add a 3.6 build for OSX also, right?

Reply:
There are no (up to date) OSX Conda packages for Mantid  so I'm not convinced this would be worthwhile.

Reply:
Are there any plans on the Mantid side to have an up-to-date OSX `conda` package?

Reply:
Probably not, Conda isn't an official distribution means for Mantid and support for it is very patchy.

The Conda OSX build has been broken for some time (most recent OSX package was from October last year) so I assume those that do maintain it don't care about it enough to fix it.

Reply:
ok, fine my me.